src/components/ListOfGoods/ListOfGoods.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+type Props = {
+  goods: string[],
+};
+
+export const ListOfGoods: React.FC<Props> = ({ goods }) => (
+  <ul>
+    {goods.map(good => (
+      <li
+        key={good}
+        data-cy="Good"
```

Comment:
let's also pass `style = {{color: good.color}}`

Reply:
Sorry, can you please specify, i don't have any color inside 'good', where do I get the color from?

Reply:
yes, goods just array of strings in this task. maybe Denys thought about another task)